learn/wavemaker-release-notes/v10-9-2.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "WaveMaker 10.9.2 - Release date: 01 November 2021"
+id: ""
+sidebar_label: "Release v10.9.2"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Bug Fixes
+
+- Fixed an issue related to the setFilter method of the CRUD variable showing console errors.
```

Comment:
Please mention the following bugg fixes as well:
https://wavemaker.atlassian.net/browse/WMS-21289
https://wavemaker.atlassian.net/browse/WMS-21344
https://wavemaker.atlassian.net/browse/WMS-20568

Reply:
Done. Added 21289, 21344, 20568 tickets.